### DISCUSSION TITLE
Toggle claim viewing

### DISCUSSION THREAD

**Discussion by Maul-PL** [GriefPrevention/GriefPrevention#787] [Ideas]
Hi

As an Admin
I want to be able to see all claims borders and be able to toggle viewing mode on/off
Because I want to know where the claims are without having to click around with a stick. 
I want to look around and know what is around without having to know exact spot or names.
Seeing all claim borders would be simple and perfect, however some kind of map with claims is also a good thing to go.

**Comment by Jikoo** [comment]
For clarity, you are aware that you can sneak while using the visualization tool to visualize all nearby claims?
